packages/grid/src/BaseCell.css

```diff
@@ -60,26 +60,26 @@
 }
 
 .salt-density-high {
-  --grid-statusIndicator-size: 6px;
+  --grid-statusIndicator-size: 5px;
 }
 
 .salt-density-medium {
-  --grid-statusIndicator-size: 8px;
+  --grid-statusIndicator-size: 7px;
 }
 
 .salt-density-low {
-  --grid-statusIndicator-size: 10px;
+  --grid-statusIndicator-size: 9px;
 }
 
 .salt-density-touch {
-  --grid-statusIndicator-size: 12px;
+  --grid-statusIndicator-size: 11px;
 }
```

Comment:
@bhoppers2008 should the above sizes be in the size foundation? trying to avoid any component specific styles

Reply:
I think this value will remain specific to this icon in particular. Mainly because Ben was trying to squeeze it in without obscuring overflowing text. We don't necessarily have to put everything in the foundations and characteristics. If it is not repeated in multiple places and we are not confident about reusability, it should remain locally defined. Bad abstractions are worse than no abstractions.